.github/workflows/CodeAnalysisAndClean.yml

```diff
@@ -14,9 +14,34 @@ jobs:
 
       - name: Run build
         run: mvn clean install
+  run-ui-tests:
```

Comment:
Can't this part stay in the `ci.yaml`?

Reply:
Renamed this to run-ui-lint and removed the ui tests from this yml. The ci.yml still has UI unit tests.